steps/python/python.deploy.yml

```diff
@@ -0,0 +1,101 @@
+# python.deploy
+#
+# Deploy Python package to PyPI and optionally to a Docker registry.
+#
+
+parameters:
+  pypiConnector: "testpypi-tomtom-dev"
+  dockerDeploy: true
+  dockerRegistryConnector: "tomtom-docker-registry-bintray"
+  dockerRepo: "python"
+  dockerDeployFile: "Dockerfile"
+
+steps:
+- checkout: self
+  # Allow scripts to access the system token so that the pipeline can push git changes (eg. version bump)
+  persistCredentials: true
+
+- script: |
+    git config --global user.email "44620811+tt-ci@users.noreply.github.com"
+    git config --global user.name "tt-ci"
+  condition: eq(variables['release'], 'true')
+  displayName: "Configure git"
+
+# Checkout branch since by default Azure Pipelines do checkouts of specific commits (detached HEAD)
+# and we need to be at master to be able to push version bump changes.
+# We need to use a powershell script here to extract the proper branch name (eg. 'feature/feat-a' out of 'refs/heads/feature/feat-a')
+- powershell: write-host "##vso[task.setvariable variable=branch]$($env:BUILD_SOURCEBRANCH.substring($env:BUILD_SOURCEBRANCH.indexOf('/', 5) + 1))"
+  condition: eq(variables['release'], 'true')
+  displayName: "Extract the branch name"
+
+- script: git checkout $(branch)
+  condition: eq(variables['release'], 'true')
+  displayName: "Checkout branch"
+
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: "3.6"
```

Comment:
I wonder if the Python version should be parameterized? In the JSL version of this pipeline, you can substitute in a Dockerfile with other Python versions, so its possible to switch them out there

Reply:
Added possibility to override the python version (default: 3.6)